src/context/auth-context.js

```diff
@@ -0,0 +1,21 @@
+import { createContext, useContext, useState } from "react";
+
+const AuthContext = createContext()
+
+const AuthContextProvider = ({children}) => {
+
+    const [email, setEmail] = useState("")
```

Comment:
Why are you keeping these details in a context. I don't see these can be reused in anywhere else. 

Ex- Confirm password only during signup only. you are using it and validating after that there is no need of it right.

Reply:
I'm sharing email and password between login and signup pages
I could've initialised the state in app.js as well but I wanted to keep my state and updates centralised in a context